src/components/reservation/helper.ts

```diff
@@ -149,4 +150,21 @@ export const getAuthorLine = (
     : [t("materialHeaderAuthorByText"), author, year].join(" ");
 };
 
+export const getManifestationsToReserve = (
+  reservableManifestations: Manifestation[] | null,
+  isPeriodical?: boolean
+) => {
+  if (isPeriodical) {
+    return reservableManifestations;
```

Comment:
Consider adding a comment explaining *why* we want to do parallel reservations for periodicals. When I think of it that does not make much sense to me. If the user wants Time Magazine February 2023, then doing a parallel reservation for another version seems odd.